### PR TITLE
Update Manage jobs doc with Failed jobs. (#235)

### LINKED ISSUE #235: Document the failed jobs tab on the Manage Jobs page doc
See https://github.com/artefactual/atom/issues/1613.
Page to be updated: https://accesstomemory.org/en/docs/2.7/user-manual/administer/manage-jobs/

### PR DESCRIPTION
**Pull request by melaniekung**

### CODE REVIEW

File: user-manual/administer/manage-jobs.rst [hunk 1]
```diff
@@ -110,16 +110,17 @@ below: :ref:`job-details`.
...
-list of jobs has not been cleared in some time. Click on the blue hyperlink to
-switch the view from "All jobs" to "Active jobs". AtoM will refresh the page and
+   :alt: An image of the Jobs page tabs
+
+The Jobs page also includes tabs that allows users to view only those jobs that
+are currently still being executed (e.g., a status of "Running") or jobs that
+have failed (e.g., a status of "Error"). These can be useful when a large number
+of jobs are being executed simultaneously, and/or the list of jobs has not been
+cleared in some time. Click on one of the other tabs to switch the view from
+"All jobs" to "Active jobs" or "Failed job". AtoM will refresh the page and
```
Comment (anvit) [inline user-manual/administer/manage-jobs.rst]: Should say "Failed jobs" instead of "Failed job"

**Review by anvit — APPROVED** [review]
Minor typo, but looks good to me otherwise!

**Review by fiver-watson — COMMENTED** [review]
Look fine for the content changed... the bigger question is whether you want this commit to update all the places where there are now images of the jobs page that don't have the Failed tab, or whether you want to wait until all images are updated as part of the BS5 documentation update as part of #238 . 

Partial list of places where I found other screenshots showing the jobs page: 

* https://www.accesstomemory.org/en/docs/2.7/admin-manual/maintenance/asynchronous-jobs/#job-and-job-scheduler-management
* https://www.accesstomemory.org/en/docs/2.7/user-manual/import-export/export-xml/#export-multiple-xml-files-using-the-clipboard
* Two on this page: https://www.accesstomemory.org/en/docs/2.7/user-manual/import-export/csv-export/
* https://www.accesstomemory.org/en/docs/2.7/user-manual/administer/settings/#cache-description-xml-exports-upon-creation-modification

There may be more but that's what I found with a first pass!

**Comment by fiver-watson** [comment]
Also, isn't this 2.8 content? This just got merged to the 2.7 branch.... might need to review a few recent PRs and make sure that only clarifications, bug fixes, etc. are added to 2.7 - new features should go to 2.8!

**Pull request merged**